ui/translations/BackendMycroft_nl.ts

```diff
@@ -0,0 +1,47 @@
+<?xml version='1.0' encoding='utf-8'?>
+<!DOCTYPE TS>
+<TS version="2.1" language="nl_NL">
+    <context>
+        <name>BackendMycroft</name>
+        <message>
+            <location line="35" filename="../BackendMycroft.qml"/>
+            <source>Requires Pairing</source>
+            <translation>Koppelen vereist</translation>
```

Comment:
@j1nx can you review these strings please ?

Reply:
That will do indeed.